main.py

```diff
@@ -1,5 +1,5 @@
 #creating a simple function
 def my_function(fname, lname):
-  print(f"{fname} {lname}")
+  print(f"Cheetas instructor: {fname} {lname}")
 
-my_function("Trenisha", "Goslee")
+my_function("jasmine", "Lopez")
```

Comment:
Please correct name.
```suggestion
my_function("Jasmine", "Lopez")
```